### PR TITLE
Seed script 

### PR DESCRIPTION
**Pull request by Lsmith2895**
@mbelesiu @JRedmon1698 @YelloFattyBean

### CODE REVIEW

**Comment by mbelesiu** [comment]
Looks good sir @Lsmith2895

File: database/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const mongoose = require('mongoose');
+const mongoUri = 'mongodb://localhost/relatedItems';
+
+const db = mongoose.connect(mongoUri);
+
+module.export = db;
```
Comment (YelloFattyBean) [inline database/index.js]: Need to add a new line under this

File: database/seeder.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+];
+
+const insertDummyProducts = () => {
+    Item.create(data)
+      //   .then( () => { console.log(data)})
+       .then(() => mongoose.disconnect())
+      .catch( (err) => console.log(err))
+};
+
+insertDummyProducts();
```
Comment (YelloFattyBean) [inline database/seeder.js:34]: same here need to add a new line after last code line

File: database/Item.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    number_of_ratings: Number,    //0-519 
+    price: Number,                //
+    onSalePrice: Number,          // number that end in .83
+    product_url: String,          // url for product image
+    onSale: Boolean,              //is the item on sale T or F
+});
+
+const Item = mongoose.model('Item', RelatedItemSchema);
+
+module.exports = Item;
```
Comment (YelloFattyBean) [inline database/Item.js:16]: Also this one needs a new line

**Review by YelloFattyBean — CHANGES_REQUESTED** [review]

**Pull request merged**